lib/html-article-parser.js

```diff
@@ -125,16 +134,18 @@ class HTMLArticleParser extends Parser {
         return [
             processImages,
             processYoutubeEmbeds,
-            processNormalizeHeaders,
-            processCleanup({}, true),
+            normalizeHeaders,
+            [ canonicalizeLinks, { baseUrl: this.parseCanonicalURI() } ],
```

Comment:
We are yet to define whether we have to canonicalize links in future pipeline, I disagree on doing it. Plus, this is not canonicalizing the URIs since only the linked article knows its canonical URI, this is a mere conversion from relative to absolute, which doesn't add anything useful down to the SDK.

I'd prefer to keep it on hold until we reach consensus.

Reply:
Fair. I think this is useful in any case.